### PR TITLE
Revamps the housemates page to use Yes/No fragment

### PR DESCRIPTION
**Pull request by bseeger**
Pivotal Tracker ticket:  #184349675

Removes unused testPage.html

Co-Authored-By: Chibuisi Enyia; cenyia@codeforamerica.org
Co-Authored-By: Cypress Borg; cborg@codeforamerica.org

### CODE REVIEW

**Comment by bseeger** [comment]
This will probably fail until the [new fragment PR](https://github.com/codeforamerica/form-flow/pull/133) is in.

File: src/main/java/org/formflowstartertemplate/app/StaticPageController.java [hunk 1]
```diff
@@ -42,8 +42,12 @@ ModelAndView getIndex(HttpSession httpSession) {
    * @return the static page template
    */
   @GetMapping("/faq")
-  String getFaq() { return "faq"; }
+  String getFaq() {
+    return "faq";
+  }
```
Comment (bseeger) [inline src/main/java/org/formflowstartertemplate/app/StaticPageController.java:48]: Java re-styled these...

**Comment by coltborg** [comment]
Hmmm the test pass locally, but fail in CI/CD...

File: src/main/resources/templates/ubi/housemates.html [hunk 2]
```diff
@@ -1,35 +1,18 @@
...
+<th:block
+    th:replace="fragments/screens/screenWithYesAndNoButtons ::
+  screenWithYesAndNoButtons(
+    title=#{housemates.housemates},
+    header=#{housemates.do-you},
+    formAction=${formAction},
+    inputName='hasHousehold',
+    inputContent=~{::inputContent})">
+  <th:block th:ref="inputContent">
+    <!-- Be sure to have `ariaLabel='header'` to label the input with the header -->
```
Comment (coltborg) [inline src/main/resources/templates/ubi/housemates.html:10]: This comment doesn't really apply here and is not needed.

```suggestion
```

**Review by coltborg — APPROVED** [review]
lgtm, left a minor comment, but not a show stopper. I assume the tests will pass once the [library pr](https://github.com/codeforamerica/form-flow/pull/133) is merged and deployed to maven central

**Pull request merged**